third_party/inputmask/index.js

```diff
@@ -0,0 +1,21 @@
+
+import {factory as inputmaskDependencyFactory} from './inputmask.dependencyLib';
+import {factory as inputmaskFactory} from './inputmask';
+import {factory as inputmaskDateExtensionsFactory} from './inputmask.date.extensions';
+
+let Inputmask;
+
+window.inputmaskFactory = function(element) {
```

Comment:
I'm not sure what the precedence is here, but you probably want to scope this under something with "AMP" in the name (e.g. under `window.AMP`?). There are niche but legitimate use cases where viewers will inject JS into the AMP document and there's risk of name collision.

Reply:
Good point, scoping is always a good choice. I'll do that